controller/test-files/deployments_osio/namespace-by-type.json

```diff
@@ -0,0 +1,44 @@
+{
+  "data": {
+    "attributes": {
+      "bio": "",
+      "cluster": "https://some-random-cluster-dot.com",
+      "company": "",
+      "contextInformation": {
+        "experimentalFeatures": {
+          "enabled": true
+        },
+        "recentContexts": [
+          {
+            "space": "00000000-0000-0000-0000-000000000000",
+            "user": "11111111-1111-1111-1111-111111111111"
+          },
+          {
+            "space": null,
+            "user": "22222222-2222-2222-2222-222222222222"
+          }
+        ],
+        "recentSpaces": ["33333333-3333-3333-3333-333333333333"]
+      },
+      "created-at": "2017-11-03T16:39:45.566361Z",
```

Comment:
When you generate golden files, please make sure that you have this at the top of your test function:

```go
resetFn := s.DisableGormCallbacks()
defer resetFn()
```

This ensures that gorm won't automatically create dates for you. That simplifies comparisons later.